src/Backend/Core/Engine/Meta.php

```diff
@@ -471,39 +478,16 @@ public function save($update = false)
         }
 
         // build meta
-        $meta['keywords'] = $keywords;
-        $meta['keywords_overwrite'] = $this->frm->getField('meta_keywords_overwrite')->getActualValue();
-        $meta['description'] = $description;
-        $meta['description_overwrite'] = $this->frm->getField('meta_description_overwrite')->getActualValue();
-        $meta['title'] = $title;
-        $meta['title_overwrite'] = $this->frm->getField('page_title_overwrite')->getActualValue();
-        $meta['url'] = $URL;
-        $meta['url_overwrite'] = $this->frm->getField('url_overwrite')->getActualValue();
-        $meta['custom'] = $custom;
-        $meta['data'] = null;
-        if ($this->frm->getField('seo_index')->getValue() != 'none') {
-            $meta['data']['seo_index'] = $this->frm->getField('seo_index')->getValue();
-        }
-        if ($this->frm->getField('seo_follow')->getValue() != 'none') {
-            $meta['data']['seo_follow'] = $this->frm->getField('seo_follow')->getValue();
-        }
-        if (isset($meta['data'])) {
-            $meta['data'] = serialize($meta['data']);
-        }
-
```

Comment:
I see that you aren't setting the $meta parameter anymore.
Where has this gone to?